requirements_dev.txt

```diff
@@ -30,7 +30,7 @@ tox
 pre-commit
 typed-ast>=1.5.0
 #black>=19.0a0,<20.0a0
-black>=22.0
+black>=23.0
```

Comment:
This should probably be black 23.*.* - not 24.* or everything will break when that version comes out (again).

Reply:
changed to black ~= 23.0